web/app/scripts/nominatim/module.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+    'use strict';
+
+    angular.module('driver.nominatim', [
```

Comment:
I hear what you're saying, but the nominatim doesn't behave like other geocoders. It seems to be geared toward matching points up with place names, like the name of a street or a building, rather than a street address, although it does that too. The technical spec references using the OSM nominatim specifically, of which I chose one of a few providers. Since what the "nominatim" does seems to be slightly different than a geocoder as we know it, I gave it a distinct name.

If you think we should think of this feature more genericly, I can rename it to geocoder.